content/reference-rippled.md

```diff
@@ -2177,8 +2179,8 @@ An example of the request format:
 *Commandline*
 
 ```
-#Syntax account_tx account [ledger_index_min [ledger_index_max [limit]]] [binary] [count] [forward]
-rippled account_tx r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 -1 -1 2 false false false
+#Syntax account_tx account ledger_index_min ledger_index_max [offset] [limit] [binary] [count] [forward]
+rippled -- account_tx r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59 -1 -1 2 5 1 0 1
```

Comment:
Commandline still requires offset. Added it back to syntax and commandline example. Added `--` to example to treat `-_x_` as a field value, not a flag. Changed Boolean true/false values to 1/0 values as commandline is not accepting true/false.

Reply:
I think it's fine not to add extra confusion for the sake of explaining the unsupported commandline interface.